CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## 5.2.19 - *2023-04-18*
```

Comment:
Please do not add a new release. This will be handled automatically by our release system.

Reply:
Oh, so put it under `## Unreleased` instead?